### PR TITLE
Document the supported kubernetes versions

### PR DESCRIPTION
**Pull request by ccfishk**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
  Clarify which Kubernetes versions are supported

**Which issue(s) this PR fixes**:

Fix https://github.com/kubernetes-sigs/gateway-api/issues/737

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: site-src/v1alpha2/faq.md [hunk 1]
```diff
@@ -61,6 +61,8 @@
   guide what revisions are needed for the next release. It is possible
   that the next release will contain breaking changes.
 
+* **Q: What are the supported kubernetes version ?<br>**
```
Comment (robscott) [inline site-src/v1alpha2/faq.md:64]:
```suggestion
* **Q: Which Kubernetes versions are supported?<br>**
```

File: site-src/v1alpha2/faq.md [hunk 2]
```diff
@@ -61,6 +61,8 @@
...
+  A: Generally, we support Kubernetes 1.16+, but certain features like AppProtocol depend on Kubernetes 1.18 (opt-in) or 1.19 (on by default). There is not any other exceptions to the 1.16+ guideline right now.
```
Comment (robscott) [inline site-src/v1alpha2/faq.md:65]:
```suggestion
  A: Generally, we support Kubernetes 1.16+, but certain features like AppProtocol depend on Kubernetes 1.18 (opt-in) or 1.19 (on by default). There are not any other exceptions to the 1.16+ guideline right now.
```

Comment (robscott) [reply site-src/v1alpha2/faq.md:65]: Do you mind wrapping these lines at 80 characters? Although it's not an absolute rule, we usually try to keep our line lengths under that length.

**Review by robscott — COMMENTED** [review]
Thanks for the work on this @ccfishk! This mostly LGTM, just a couple nits. Looks like you'll also need to sign the CLA before this can get in.

**Comment by ccfishk** [comment]
> Welcome @ccfishk!
> 
> It looks like this is your first PR to [kubernetes-sigs/gateway-api](https://github.com/kubernetes-sigs/gateway-api) 🎉. Please refer to our [pull request process documentation](https://git.k8s.io/community/contributors/guide/pull-requests.md) to help your PR have a smooth ride to approval.
> 
> You will be prompted by a bot to use commands during the review process. Do not be afraid to follow the prompts! It is okay to experiment. [Here is the bot commands documentation](https://go.k8s.io/bot-commands).
> 
> You can also check if kubernetes-sigs/gateway-api has [its own contribution guidelines](https://github.com/kubernetes-sigs/gateway-api/tree/master/CONTRIBUTING.md).
> 
> You may want to refer to our [testing guide](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md) if you run into trouble with your tests not passing.
> 
> If you are having difficulty getting your pull request seen, please follow the [recommended escalation practices](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#why-is-my-pull-request-not-getting-reviewed). Also, for tips and tricks in the contribution process you may want to read the [Kubernetes contributor cheat sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet/README.md). We want to make sure your contribution gets all the attention it needs!
> 
> Thank you, and welcome to Kubernetes. 😃

@hbagdi  kindly point how should I proceed to resolve this.

**Comment by ccfishk** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
> * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)

I already signed the document successfully.

You have successfully signed your document |  
-- | --

**Comment by ccfishk** [comment]
"I signed it"

**Comment by robscott** [comment]
@ccfishk Yep, I think that should fix it.

**Comment by ccfishk** [comment]
"I signed it".  And change the email account to primary email account.

**Comment by hbagdi** [comment]
@ccfishk You have created the commits on this PR with two separate email addresses:

```
commit 03197d581ff9ded667acfd58d9004a5fd8c1f5db (HEAD -> 755, pr/755)
Author: ccfishk <jimin.hu@konghq.com>
Date:   Tue Aug 10 17:51:22 2021 -0700

    fix email address

commit 28353e4a25a218631544a979131533a709619ab1
Author: ccfishk <jimin.susan.hu@gmail.com>
Date:   Mon Aug 9 15:37:46 2021 -0700

    format restrict each line less than 80 characters.

commit 4e78d449085f09081dbe96b3c384777d4f52ceb6
Author: Jimin <jimin.susan.hu@gmail.com>
Date:   Mon Aug 9 15:34:09 2021 -0700

    Update site-src/v1alpha2/faq.md
    
    Co-authored-by: Rob Scott <rob.scott87@gmail.com>

commit 27840814f3b7d04819c38ec16dcca51f20986c69
Author: Jimin <jimin.susan.hu@gmail.com>
Date:   Mon Aug 9 15:30:19 2021 -0700

    Update site-src/v1alpha2/faq.md
    
    Co-authored-by: Rob Scott <rob.scott87@gmail.com>

commit a74bfdeb5a66ada7c35f7d2412e56b1408b7adb4
Author: ccfishk <jimin.susan.hu@gmail.com>
Date:   Mon Aug 9 07:04:28 2021 -0700

    Fix kubernetes version support

```
I'd recommend squashing these commits and then using the same email address you used to sign the CLA to use in the commit.
Using the same email address as the CLA will be required for any subsequent contribution as well.

**Comment by ccfishk** [comment]
> @ccfishk You have created the commits on this PR with two separate email addresses:
> 
> ```
> commit 03197d581ff9ded667acfd58d9004a5fd8c1f5db (HEAD -> 755, pr/755)
> Author: ccfishk <jimin.hu@konghq.com>
> Date:   Tue Aug 10 17:51:22 2021 -0700
> 
>     fix email address
> 
> commit 28353e4a25a218631544a979131533a709619ab1
> Author: ccfishk <jimin.susan.hu@gmail.com>
> Date:   Mon Aug 9 15:37:46 2021 -0700
> 
>     format restrict each line less than 80 characters.
> 
> commit 4e78d449085f09081dbe96b3c384777d4f52ceb6
> Author: Jimin <jimin.susan.hu@gmail.com>
> Date:   Mon Aug 9 15:34:09 2021 -0700
> 
>     Update site-src/v1alpha2/faq.md
>     
>     Co-authored-by: Rob Scott <rob.scott87@gmail.com>
> 
> commit 27840814f3b7d04819c38ec16dcca51f20986c69
> Author: Jimin <jimin.susan.hu@gmail.com>
> Date:   Mon Aug 9 15:30:19 2021 -0700
> 
>     Update site-src/v1alpha2/faq.md
>     
>     Co-authored-by: Rob Scott <rob.scott87@gmail.com>
> 
> commit a74bfdeb5a66ada7c35f7d2412e56b1408b7adb4
> Author: ccfishk <jimin.susan.hu@gmail.com>
> Date:   Mon Aug 9 07:04:28 2021 -0700
> 
>     Fix kubernetes version support
> ```
> 
> I'd recommend squashing these commits and then using the same email address you used to sign the CLA to use in the commit.
> Using the same email address as the CLA will be required for any subsequent contribution as well.

As a verify fix, I opened a new PR https://github.com/kubernetes-sigs/gateway-api/pull/769 which pushed using the same email where receiving the "HelloSign" confirmation.

**Comment by youngnick** [comment]
/lgtm

Congrats on getting the CLA stuff sorted out!

**Comment by robscott** [comment]
Thanks @ccfishk! 

/approve

**Pull request merged**